imod/mf6/mf6_wel_adapter.py

```diff
@@ -53,11 +54,13 @@ def __init__(
         rate,
         concentration=None,
         concentration_boundary_type="aux",
+        save_flows: Optional[bool] = None,
         validate: bool = True,
     ):
         super().__init__()
         self.dataset["cellid"] = cellid
         self.dataset["rate"] = rate
+        self.dataset["save_flows"] = save_flows
```

Comment:
Nice, related to this issue I think: https://github.com/Deltares/imod-python/issues/652